test/test_automated_install.py

```diff
@@ -293,6 +294,79 @@ def test_installPiholeWeb_fresh_install_no_errors(Pihole):
     assert 'blockingpage.css' in web_directory
 
 
+def test_installPiholeWeb_fresh_install_readableBlockpage(Pihole):
+    '''
+    confirms all web page assets from Core repo are installed and readable by www-data on a fresh build
+    '''
+    installWeb = Pihole.run('''
+    umask 0027
+    source /opt/pihole/basic-install.sh
+    installPiholeWeb
+    ''')
+    expected_stdout = info_box + ' Installing blocking page...'
+    assert expected_stdout in installWeb.stdout
+    expected_stdout = tick_box + (' Creating directory for blocking page, '
+                                  'and copying files')
+    assert expected_stdout in installWeb.stdout
+    expected_stdout = info_box + ' Backing up index.lighttpd.html'
+    assert expected_stdout in installWeb.stdout
+    expected_stdout = ('No default index.lighttpd.html file found... '
+                       'not backing up')
+    assert expected_stdout in installWeb.stdout
+    expected_stdout = tick_box + ' Installing sudoer file'
+    assert expected_stdout in installWeb.stdout
+    exit_status_success = '0'
+    # check directories above $webroot for read and execute permission by www-data
+    check_var = 'which sudo'
+    print (Pihole.run(check_var).stdout)
+    check_var = 'which test'
+    print (Pihole.run(check_var).stdout)
+    check_var = 'test -r /var'
+    print (Pihole.run(check_var).stdout)
+    check_var = 'echo $(test -r /var) $?'
+    print (Pihole.run(check_var).stdout)
+    check_var = 'sudo -u www-data test -r /var'
```

Comment:
```suggestion
    check_var = 'su  -c "test -r /var/" www-data'
```

Reply:
Something along those lines, may need more `su` flags but the man is pretty good.

```sh
root@b5297e950e96:/# su --help
Usage: su [options] [LOGIN]

Options:
  -c, --command COMMAND         pass COMMAND to the invoked shell
  -h, --help                    display this help message and exit
  -, -l, --login                make the shell a login shell
  -m, -p,
  --preserve-environment        do not reset environment variables, and
                                keep the same shell
  -s, --shell SHELL             use SHELL instead of the default in passwd
```